tests/tests.py

```diff
@@ -257,7 +260,7 @@ def test_elb(self):
             ProjectConfig(
                 'tests/sample-project.yaml',
                 'dev').config,'my-stack-name')
-        self.assertEquals(known, config.elb())
+        compare(known, config.elb()[0])
```

Comment:
This looks like a suspect – can you assidn `elb, elb_sg` and then compare elb directly as it's clearer what this is doing.